### DISCUSSION TITLE
alist webdav 用rclone挂载到本地后 怎么取消挂载

### DISCUSSION THREAD

**Discussion by Yumega** [AlistGo/alist#5123] [Q&A] [answered]
挂载位置 是 /mnt/sdc1/Alidisk/

怎么取消挂载，换个位置挂载

**Comment by Troray** [comment]
> 挂载位置 是 /mnt/sdc1/Alidisk/
> 
> 怎么取消挂载，换个位置挂载

`fusermount -u /mnt/sdc1/Alidisk`

**Comment by Yumega** [comment, marked as answer]
> > 挂载位置 是 /mnt/sdc1/Alidisk/
> > 怎么取消挂载，换个位置挂载
> 
> `fusermount -u /mnt/sdc1/Alidisk`

Thank you, 问题已解决
